src/components/Filter/__mocks__/mockFilterExample.js

```diff
@@ -0,0 +1,542 @@
+import React from 'react';
+import {
+  Grid,
+  Col,
+  Row,
+  Filter,
+  FilterTypeSelector,
+  FilterValueSelector,
+  FilterCategorySelector,
+  FilterCategoryValueSelector
+} from '../../../index';
+import { boolean } from '@storybook/addon-knobs/dist/index';
```

Comment:
This is throwing an error for me in the console...

Suggestion is to move the boolean up to the `Filter.stories.js` file and pass it down as a prop to the `MockFilterExample`. I tend to define all the Knobs on the `*.stories.js` file, but that is just preference...makes it easier to copy/paste those imports...

Reply:
I've removed the need for the import.